test/boost/s3_test.cc

```diff
@@ -54,21 +54,37 @@ s3::endpoint_config_ptr make_minio_config() {
             .region = ::getenv("AWS_DEFAULT_REGION") ? : "local",
         }},
     };
-    return make_lw_shared<s3::endpoint_config>(std::move(cfg));
+    return {tests::getenv_safe("PROXY_S3_SERVER_HOST"), make_lw_shared<s3::endpoint_config>(std::move(cfg))};
 }
 
+std::tuple<std::string,s3::endpoint_config_ptr> make_minio_config() {
```

Comment:
also, how about returning a `s3::client` ? simpler this way. as nobody cares about `host` and `s3::endpoint_config_ptr` apart from `s3::client::make()`.

Reply:
Done